src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -133,6 +134,8 @@ class CompiledModel : public ov::ICompiledModel {
     std::string m_ref_device;
 
     execution_stats m_total_stat;
+
+    std::string m_weights_bank_name = "";
```

Comment:
so with the above comment(s) in mind here we should have a bank object, not its name

Reply:
Done